trace-protobuf/build.rs

```diff
@@ -68,16 +66,7 @@ fn generate_protobuf() {
         "#[serde(rename_all = \"PascalCase\")]",
     );
 
-    config.field_attribute("ClientStatsPayload.agentAggregation", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.env", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.service", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.containerID", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.tags", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.version", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.lang", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.tracerVersion", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.runtimeID", "#[serde(default)]");
-    config.field_attribute("ClientStatsPayload.sequence", "#[serde(default)]");
+    config.field_attribute(".pb.ClientStatsPayload", "#[serde(default)]");
```

Comment:
if any fields in `ClientStatsPayload` don't exist during deserialization, give them the default value.